src/osemgrep/cli_interactive/Framed_zipper.mli

```diff
@@ -43,6 +43,7 @@ type 'a t
 val move_left : 'a t -> 'a t
```

Comment:
> A zipper is, of course, the classic data structure for a
>   list's "one hole context", of traversing a list via being located
 >  at a single point.

Remove the 'of course', and add a link to a web page explaining what a zipper is.